app/routes/pages/components/PageDetail.js

```diff
@@ -95,8 +97,11 @@ function PageDetail<T: Object>({
         )}
       </NavigationTab>
       <Flex className={styles.page} wrap>
-        <PageRenderer page={selectedPage} />
-
+        {isComplete ? (
```

Comment:
Did this because `LoadingIndicator wrapps it's content children in a div, and I wan't no div.